seed/views/portfoliomanager.py

```diff
@@ -82,7 +82,15 @@ def report(self, request):
         template = request.data['template']
         pm = PortfolioManagerImport(username, password)
         try:
-            content = pm.generate_and_download_template_report(template)
+            if 'z_seed_child_row' not in template:
+                return JsonResponse(
+                    {'status': 'error', 'message': 'Invalid template formulation during portfolio manager data import'},
+                    status=status.HTTP_400_BAD_REQUEST
+                )
+            if template['z_seed_child_row']:
+                content = pm.generate_and_download_child_data_request_report(template)
+            else:
+                content = pm.generate_and_download_template_report(template)
```

Comment:
"Child" data request rows, and "parent" template rows behave significantly different, so we have a switch here on which worker function to call to actually get the XML data response.